packages/google/src/google-generative-ai-settings.ts

```diff
@@ -55,6 +55,15 @@ Optional. A list of unique safety settings for blocking unsafe content.
       | 'BLOCK_ONLY_HIGH'
       | 'BLOCK_NONE';
   }>;
+  /**
+   * Optional. Enables timestamp understanding for audio-only files.
+   * This is a preview feature.
+   *
+   * Available for the following models:
```

Comment:
Instead of listing the supported models here, is there a page we can link to in vertex docs?